README.md

```diff
@@ -1,3 +1,13 @@
 # TuxxChatApp
 
+To get started with the **TuxxChatApp**:
+
+1. Fork and clone down this repo.
+2. Run `npm install`
+3. Run `npm start` to begin a `watchify` build of the app
```

Comment:
Maybe we should explain the fact that npm start automatically updates any changes you make into build.js? if i was a first time user i would have no idea what a watchify build does ...